docs/api/en/core/BufferGeometry.html

```diff
@@ -144,6 +144,14 @@ <h3>[property:Object morphAttributes]</h3>
 			Hashmap of [page:BufferAttribute]s holding details of the geometry's [page:Geometry.morphTargets morphTargets].
 		</p>
 
+		<h3>[property:Boolean morphTargetsRelative]</h3>
+		<p>
+			Used to control the morph target behavior; when set to true, the morph target data is treated as relative changes to the base geometry,
+			instead of completely replacing the base geometry data.
```

Comment:
When set to true, the morph target data is treated as relative offsets, rather than as absolute positions.

Reply:
Thanks! This wording is much better; I've adjusted this to say "positions/normals".